src/clouds.cpp

```diff
@@ -58,13 +57,13 @@ Clouds::Clouds(scene::ISceneManager* mgr,
 	//m_material.MaterialType = video::EMT_TRANSPARENT_VERTEX_ALPHA;
 	m_material.MaterialType = video::EMT_TRANSPARENT_ALPHA_CHANNEL;
 
+	m_params.height        = cloudheight ? cloudheight : 120;
```

Comment:
Um, no, it is used. It's the height of the clouds in the main menu. Try it: change the line I linked above to "200", or "20", and start Minetest.

Since removing that would mean "let's change the look of the main menu for everyone", I think it's out of scope for this PR.